tofu/modules/pya/main.tf

```diff
@@ -284,3 +284,13 @@ resource "aws_cloudwatch_log_subscription_filter" "datadog" {
   filter_pattern  = ""
   destination_arn = data.aws_lambda_function.datadog["this"].arn
 }
+
+module "cloudfront_waf" {
```

Comment:
I couldn’t find a way to reference the logging module from the main.tf, and so i had to add this to the shared module. i _think_ this is okay since we don’t have automatic deploys to prod set up. so i will test this on staging first and then deploy to prod.